app/routes/api/v1/warehouse/orders/index.ts

```diff
@@ -0,0 +1,9 @@
+import { Order } from "@prisma/client";
+import { LoaderFunction } from "@remix-run/node";
+import { findOrders } from "~/models/order/order.server";
+
+type LoaderData = Order[];
+
+export const loader: LoaderFunction = async (): Promise<LoaderData> => {
+  return await findOrders();
+};
```

Comment:
Is this needed?

Reply:
This is definitely needed as it is the loader that returns every order in the DB

Reply:
whops, my bad